health.go

```diff
@@ -13,7 +13,7 @@ type Health interface {
 	// RegisterCheck registers a health check according to the given configuration.
 	// Once RegisterCheck() is called, the check is scheduled to run in it's own goroutine.
 	// Callers must make sure the checks complete at a reasonable time frame, or the next execution will delay.
-	RegisterCheck(cfg *Config) error
+	RegisterCheck(check Check, opts ...CheckOption) error
```

Comment:
You need to fix the README for this change

Reply:
done